src/services/dumper/agent-nodejs-dumper.ts

```diff
@@ -0,0 +1,118 @@
+import type { ConfigInterface } from '../../interfaces/project-create-interface';
+
+import toValidPackageName from '../../utils/to-valid-package-name';
+import AbstractDumper from './abstract-dumper';
+
+export default class AgentNodeJsDumper extends AbstractDumper {
+  private env: any;
+
+  private readonly DEFAULT_PORT = 3310;
+
+  private readonly isLinuxOs: any;
+
+  constructor(context) {
+    const { assertPresent, env, isLinuxOs } = context;
+
+    assertPresent({
+      env,
+      isLinuxOs,
+    });
+
+    super(context);
+
+    this.env = env;
+    this.isLinuxOs = isLinuxOs;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  get templateFolder() {
+    return 'agent-nodejs/';
+  }
+
+  writePackageJson(dbDialect: string, applicationName: string) {
+    const dependencies: any = {
+      nodenv: '^16.0.1',
+      '@forestadmin/agent': '^1.0.0',
+    };
+
+    if (dbDialect === 'mongodb') {
+      dependencies['@forestadmin/datasource-mongoose'] = '^1.0.0';
+    } else {
+      dependencies['@forestadmin/datasource-sql'] = '^1.0.0';
+    }
+
+    if (dbDialect) {
+      if (dbDialect.includes('postgres')) {
+        dependencies.pg = '~8.2.2';
+      } else if (dbDialect === 'mysql') {
+        dependencies.mysql2 = '~2.2.5';
+      } else if (dbDialect === 'mssql') {
+        dependencies.tedious = '^6.4.0';
+      }
+    }
+
+    const pkg = {
+      name: toValidPackageName(applicationName),
+      version: '0.0.1',
+      private: true,
+      scripts: { start: 'node ./index.js' },
```

Comment:
we might want to consider nodemon by default here

Reply:
Yep !